osu.Server.Queues.ScoreStatisticsProcessor/Processors/ScorePerformanceProcessor.cs

```diff
@@ -191,6 +191,10 @@ public static bool AllModsValidForPerformance(Mod[] mods)
 
                         continue;
 
+                    case ModClassic:
+                        // Classic mod is only allowed if it's attached to legacy scores (null build ID).
+                        return score.BuildID == null;
```

Comment:
Isn't examining that `LegacyScoreID` is not null the standard / canonical way to check if a `SoloScoreInfo` is legacy or not? I worry about introducing multiple heuristics for the same thing.

Reply:
Hmmm...

https://github.com/ppy/osu-queue-score-statistics/blob/3def49348ed93535df38d4bc8ffdbdb94709df47/osu.Server.Queues.ScoreStatisticsProcessor/Processors/UserTotalPerformanceProcessor.cs#L119-L124

@peppy any opinions?

Reply:
Just for context, I'm referring to

https://github.com/ppy/osu-queue-score-statistics/blob/3def49348ed93535df38d4bc8ffdbdb94709df47/osu.Server.Queues.ScoreStatisticsProcessor/ScoreStatisticsQueueProcessor.cs#L101-L105
https://github.com/ppy/osu-queue-score-statistics/blob/3def49348ed93535df38d4bc8ffdbdb94709df47/osu.Server.Queues.ScoreStatisticsProcessor/Commands/Queue/DeleteImportedHighScoresCommand.cs#L78-L79

as places that use the other method.

Reply:
Will wait on a new osu! nuget package before fixing this.